bot/code_review_bot/report/base.py

```diff
@@ -97,27 +80,38 @@ def requires(self, configuration, *keys):
     def calc_stats(self, issues):
         """
         Calc stats about issues:
-        * group issues by class name
+        * group issues by analyzer
         * count their total number
         * count their publishable number
         """
-        groups = itertools.groupby(
-            sorted(issues, key=lambda x: str(x.__class__)), lambda x: x.__class__
-        )
 
-        def stats(items):
+        def by_analyzer(i):
+            return i.analyzer
+
+        groups = itertools.groupby(sorted(issues, key=by_analyzer), by_analyzer)
+
+        def stats(analyzer, items):
             _items = list(items)
+
+            # Lookup the help message, supporting regexes
+            _help = HELP_COMMANDS.get(analyzer)
+            if _help is None:
+                for regex, help_value in HELP_COMMANDS.items():
+                    if isinstance(regex, Pattern) and regex.match(analyzer):
+                        _help = help_value
+                        break
+
             return {
+                "analyzer": analyzer,
```

Comment:
We could remove the `source-test-` part when analyzer starts with `source-test-` to have a cleaner name in the comment

Reply:
(here or in `build_comment` directly)